actionpack/test/dispatch/request_id_test.rb

```diff
@@ -8,7 +8,7 @@ class RequestIdTest < ActiveSupport::TestCase
   end
 
   test "ensure that only alphanumeric uurids are accepted" do
-    assert_equal "X-Hacked-HeaderStuff", stub_request("HTTP_X_REQUEST_ID" => "; X-Hacked-Header: Stuff").request_id
+    assert_equal "X-Hacked-HeaderStuff@", stub_request("HTTP_X_REQUEST_ID" => ";\r\n X-Hacked-Header: Stuff@").request_id
```

Comment:
Let's be a little more explicit about this, and add a new test that uses an actual Apache mod_request_id value, and labels it as such. Otherwise `@` seems a pretty arbitrary extension to the alphabet needed to cover UUIDs.